frontend/public/components/cluster-settings/global-config.tsx

```diff
@@ -41,9 +46,14 @@ class GlobalConfigPage_ extends React.Component<GlobalConfigPageProps, GlobalCon
           return [];
         }).then(items => items.map((i: K8sKind) => ({...i, model})));
     })).then((responses) => {
-      const items = _.sortBy(_.flatten(responses), 'kind', 'asc');
+      const knativeConfig = this.getKnativeConfig();
+      const flattenedResponses = _.flatten(responses);
+      const winnowedResponses = flattenedResponses.map(item => ({model: item.model, uid: item.metadata.uid, name: item.metadata.name, namespace: item.metadata.namespace, kind: item.kind}));
+      const allResponses = knativeConfig[0].properties.componentProps.required === 'KNATIVE_SERVING' ? winnowedResponses.concat({model: knativeConfig[0].properties.componentProps.model, uid: 'knative-serving', name: 'knative-serving', namespace: 'knative-serving', kind: 'KnativeServing'}) : winnowedResponses;
```

Comment:
This is too knative specific when this component should be generic. Update the extension to contain all the missing information (looks like the `uid` and `name` and `namespace` are missing). Then merge the extensions with the `configResources`.

Also don't you need to check the flags before including this?

Reply:
Is checking the flag supposed to be done through getFeatureFlags() in registry.ts?

Reply:
You'll have to pull the flags from redux and check them at render time.

Reply:
Thanks, I'll work on this.